### PR TITLE
ac: add Polish translation

### PR DESCRIPTION
**Pull request by rozie**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by sebastiaanspeck — APPROVED** [review]
LGTM!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Note to new maintainers:** Wait for an approving review from a native speaker (code owner) before merging.

File: pages.pl/linux/ac.md
Comment (acuteenvy) [inline pages.pl/linux/ac.md:24]:
```suggestion
> Wyświetl statystyki dotyczące czasu połączenia użytkowników.
> Więcej informacji: <https://www.gnu.org/software/acct/manual/accounting.html#ac>.

- Wyświetl w godzinach jak długo aktualny użytkownik był połączony:

`ac`

- Wyświetl ile godzin użytkownicy byli połączeni:

`ac --individual-totals`

- Wyświetl ile godzin konkretny użytkownik był połączony:

`ac --individual-totals {{użytkownik}}`

- Wyświetl ile godzin na dzień konkretny użytkownik był podłączony (z podsumowaniem):

`ac --daily-totals --individual-totals {{użytkownik}}`

- Wyświetlaj także dodatkowe szczegóły:

`ac --compatibility`
```

**Review by acuteenvy — CHANGES_REQUESTED** [review]
Same as #11059

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**
